packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -97,6 +102,7 @@ function patchWithErrorCode(
 export function normalizeError(
     error: unknown,
     annotations: IFluidErrorAnnotations = {},
+    fluidErrorCode = "", // Issue #8472: should be required
```

Comment:
I intentionally removed this capability.  See https://github.com/microsoft/FluidFramework/pull/8124

Reply:
Added an important comment to https://github.com/microsoft/FluidFramework/issues/8472#issuecomment-988307732